### PR TITLE
Add devices page and various design updates

### PR DESCRIPTION
**Pull request by elv-zenia**
Add Compatible Devices page and make design changes per doc shared by the design team.

### CODE REVIEW

File: src/main/pages/main/CompatibleDevices.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    }
+  };
+
+  return (
+    <div className="compatible-devices__autocomplete-results">
+      {
+        (!results.length || hasMatch) ?
+          <div className="compatible-devices__autocomplete-results">
+            <h3 className="compatible-devices__autocomplete-results-header">
+              {ResultsHeader()}
```
Comment (elv-kevin) [inline src/main/pages/main/CompatibleDevices.js:35]: why define this as a separate function?

File: package.json [hunk 2]
```diff
@@ -75,6 +75,7 @@
     "html-webpack-inline-source-plugin": "0.0.10",
     "html-webpack-plugin": "^3.2.0",
     "image-webpack-loader": "^4.6.0",
+    "papaparse": "^5.4.1",
```
[the remark this answers is not in the document]
Comment (elv-kevin) [reply package.json:78]: oh, I see we provide a link to the csv, nevermind

File: src/main/pages/main/CompatibleDevices.js (hunk 1 above)
Comment (elv-zenia) [reply src/main/pages/main/CompatibleDevices.js:35]: ResultsHeader() was bigger before.. I'll remove the separation

**Comment by elv-kevin** [comment]
I think this looks good, pending design approval

**Pull request merged**